docs/user/automatic-redirects.rst

```diff
@@ -82,63 +51,40 @@ you most likely want users clicking on incoming links to see the latest version
 Good practice ✅
 ~~~~~~~~~~~~~~~~
 
-* Use ``/page/path/to/page.html`` if you are linking to the page in the default version of the default language.
-* Use ``/en/latest/path/to/page.html`` if you want to be specific about the language.
-* If you move a page that likely has incoming references, :doc:`create a redirect rule </user-defined-redirects>`.
+* Use `page redirects <user-defined-redirects:Page redirects>`_ if you are linking to the page in the default version of the default language.
+* If you move a page that likely has incoming references, :doc:`create a custom redirect rule </guides/redirects>`.
 * Links from other Sphinx projects should use :doc:`intersphinx </guides/intersphinx>`.
-* Use minimal page slugs that do not attract renaming suggestions.
+* Use minimal page :term:`slugs <slug>` that do not attract renaming suggestions.
 * When possible,
   keep original file names rather than going for low-impact URL renaming.
   Renaming an article's title is great for the reader and great for SEO,
   but this does not have to involve the URL.
+* Try to get your understanding of ``latest`` and *default version* right from the beginning and ensure you don't change them later on.
+* Keep development versions hidden so people do not find them on search engines by mistake.
+  Publishing documentation at the wrong URL can be costly if the URL starts circulating.
+* Use a :ref:`version warning <versions:Version warning>` to ensure the reader is aware in case they are reading an old (archived) version.
+
+.. tip::
+
+   Using Sphinx?
+     If you are using ``:ref:`` for :doc:`cross-referencing </guides/cross-referencing-with-sphinx>`, you may add as many reference labels to a headline as you need,
+     keeping old reference labels. This will make refactoring documentation easier and avoid that external projects
+     referencing your documentation through :doc:`intersphinx </guides/intersphinx>` break.
 
 Questionable practice 🟡
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-* Avoid using specific versions in links unless you really mean that users should see that exact version.
+* Avoid using specific versions in links unless you really intend users to see that exact version.
```

Comment:
I think the sentence is a bit more cryptic now, but it's shorter :+1:

Reply:
Not sure I'd say it's more cryptic, and not sure if this response is sarcasm?

Reply:
No, it wasn't. When text is trimmed out, *some* meaning can be lost. In this case, it's about clarifying and spelling out *unless you really intend users to see that exact version*. Because the context is a list of "Questionable practice 🟡", then the idea of spelling out why the practice can be questionable is a bit more in the spotlight :)

When shortening to *unless users need that exact version*, there's the risk that someone doesn't get the meaning."Yes, they should see that version". The overly long sentence was supposed to capture the afterthought.

Maybe not successful, I get that :+1: